otx/algorithms/detection/adapters/mmdet/utils/config_utils.py

```diff
@@ -333,13 +334,10 @@ def patch_tiling(config, hparams, dataset=None):
     if hparams.tiling_parameters.enable_tiling:
         logger.info("Tiling enabled")
 
+        # FIXME: TRAIN only?
+        # if dataset and dataset.purpose == DatasetPurpose.TRAINING and hparams.tiling_parameters.enable_adaptive_params:
```

Comment:
@eugene123tw , maybe we need to decide to what we use.
IMO, commented line is prefer option.